tools.yaml

```diff
@@ -179,6 +179,38 @@ tools:
   - name: 'cutadapt'
     owner: 'lparsons'
     tool_panel_section_label: 'FASTA/FASTQ'
+    
+   - name: 'minimus2'
```

Comment:
This will almost certainly crash or not run. The wrapper is from 2011 and doesn't include any dependencies, so, minimus2 will not be automatically installed. Minimus2 itself hasn't been touched since 2013. 

Do you know that you need this one?

Reply:
If it hasn't been touched, we won't touch it either ;)